Rakefile

```diff
@@ -132,9 +132,15 @@ task :clean do
   sh "git clean -Xdf"
 end
 
+desc "Test OnDemand"
+task :test => 'test:all'
```

Comment:
Also a good time at the bottom to add

```
task default: %w[test]
```

so the default task is the test task (which is the convention in ruby apps)